packages/flame/test/_assets/images.dart

```diff
@@ -0,0 +1,377 @@
+import 'dart:ui';
+
+import 'package:flame/assets.dart';
+
+const flameImageEncoded = '''
```

Comment:
cant we use image files on tests?

Reply:
It should be possible but I don't know how. Any suggestions?

Reply:
We are using images for goldens at work, I'll look up how we do it there.

Reply:
Maybe this will work: https://stackoverflow.com/questions/45780255 ?